apps/admin/src/app/(api)/api/v1/token/route.ts

```diff
@@ -40,13 +40,18 @@ export const POST = async (req: NextRequest) => {
             }
 
             if ((sensorData.needsScript || data.needScript) && sensorData.script) {
+                let perRotation: number | undefined = undefined;
+                if (sensorData.script.split("\n").length === 1 && sensorData.script.match(/^\d+$/)) {
+                    perRotation = Number.parseInt(sensorData.script);
+                }
                 waitUntil(log("sensor/sent-script", "info", "sensor-token", "api", { sensorData, data }));
                 return new NextResponse(
                     TokenResponse.toBinary({
                         status: 200,
                         accessToken: code,
                         expiresIn: 3600,
                         script: sensorData.script,
```

Comment:
Würde er hier nicht auch den Zahlenwert nun in das Feld für das Skript eintragen? Wenn ja, könnte man das nicht durch einen Kontrollblock o. dgl. verhindern. So könnte man den Datentransfer des einen oder anderen unnötigen Bytes verhindern.